src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java

```diff
@@ -171,6 +175,14 @@ private void updateRestrictedFlag(Map<String, Object> caseDataUpdated, boolean i
         }
     }
 
+    public boolean checkIfUserIsCourtStaff(String authorisation, CallbackRequest callbackRequest) {
```

Comment:
Hi @khurshedk,
This looks clumsy as we are checking 2 conditions, also this was giving error always 
1. upload documents with court staff/other users
2. upload with Court option selected or not.

This has been refactored in base branch, could you please take a look & let me know if your feedback.